meta-mender-core/recipes-mender/mender-client/mender-client.inc

```diff
@@ -111,7 +111,7 @@ PACKAGECONFIG_append = "${@bb.utils.contains('MENDER_FEATURES', 'mender-client-i
 PACKAGECONFIG_append = "${@bb.utils.contains('MENDER_FEATURES', 'mender-uboot', ' u-boot', '', d)}"
 PACKAGECONFIG_append = "${@bb.utils.contains('MENDER_FEATURES', 'mender-grub', ' grub', '', d)}"
 
-PACKAGECONFIG[mender-client-install] = ",,,mender-artifact-info ca-certificates"
+PACKAGECONFIG[mender-client-install] = ",,,ca-certificates"
```

Comment:
I opted for removing the standard packageconfig, and having an explicit `RDEPENDS` in the old client recipe.

Not sure whether this is _the_ way or not...

Reply:
I believe it is for the extremely theoretical case where someone wants to install the Mender client, but without all the supporting stuff around it (which is enabled by the `mender-client-install` packageconfig). But honestly it was never well defined, nor documented anywhere, so no problem to remove it.